### PR TITLE
Update Grafana ClickHouse alerting

### PR DESCRIPTION
**Pull request by Aquatoid87**
Hi, @andyone , we have updated ClickHouse alerting for Grafana, it will be nice to check this PR, and to build RPM-package from spec.

### CODE REVIEW

**Comment by gongled** [comment]
@andyone, keep in mind, that @Aquatoid87 also put systemd unit file to change `Restart` parameter from `on-failure` (by default) to `always` to let process automatically restart after database connection loss.

**Comment by andyone** [comment]
@Aquatoid87 Thanks for your contribution.

@gongled Yep, I have noticed that. I think if we set `Restart` to `always`, `StartLimitIntervalSec` and `StartLimitBurst` also should be set.

File: specs/grafana/SOURCES/grafana-server.service [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Wants=network-online.target
+After=network-online.target
+After=postgresql.service mariadb.service mysqld.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/grafana-server
+User=grafana
+Group=grafana
+Type=notify
+Restart=always
```
Comment (andyone) [inline specs/grafana/SOURCES/grafana-server.service:13]: Please add `StartLimitIntervalSec` and `StartLimitBurst`.

File: specs/grafana/SOURCES/grafana-server.service [hunk 2]
```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Grafana instance
+Documentation=http://docs.grafana.org
+Wants=network-online.target
+After=network-online.target
+After=postgresql.service mariadb.service mysqld.service
```
Comment (andyone) [inline specs/grafana/SOURCES/grafana-server.service]: Maybe we should add `clickhouse-server.service` here?

**Review by andyone — CHANGES_REQUESTED** [review]

File: specs/grafana/SOURCES/grafana-server.service (hunk 2 above)
Comment (gongled) [reply specs/grafana/SOURCES/grafana-server.service]: Fixed (4ab1291589772c484293d167c0937ffef72612fc)

File: specs/grafana/SOURCES/grafana-server.service (hunk 1 above)
Comment (gongled) [reply specs/grafana/SOURCES/grafana-server.service:13]: Fixed (4ab1291589772c484293d167c0937ffef72612fc)

**Review by andyone — APPROVED** [review]

**Comment by andyone** [comment]
Looks good for me now. 

@gongled  @Aquatoid87 Pay attention that you should look after your patches for this package. All outdated patches will be removed at the moment of the package update.

**Pull request merged**